applications/acdc/src/acdc_agent_fsm.erl

```diff
@@ -936,10 +936,10 @@ answered({'sync_req', JObj}, #state{agent_listener=AgentListener
     {'next_state', 'answered', State};
 answered({'channel_unbridged', CallId}, #state{member_call_id=CallId}=State) ->
     lager:info("caller channel ~s unbridged", [CallId]),
-    {'next_state', 'wrapup', State#state{wrapup_ref=wrapup_timer(State)}};
+    {'next_state', 'answered', State};
 answered({'channel_unbridged', CallId}, #state{agent_call_id=CallId}=State) ->
     lager:info("agent channel unbridged"),
-    {'next_state', 'wrapup', State#state{wrapup_ref=hangup_call(State)}};
+    {'next_state', 'answered', State};
```

Comment:
@danielfinke 
This and the diff above do not seem related to tracking the hangup unless I misunderstood something (possible as I don't know ACDc that well).  Are you sure that the transition here should not go to wrapup?  I suspect this might be a fix for something else ;)